k8s/upgrade/src/upgrade_resources/upgrade.rs

```diff
@@ -97,6 +98,10 @@ pub struct UpgradeArgs {
     /// Upgrade to the specified branch.
     #[clap(global = true, hide = true, long, short = 'u')]
     pub upgrade_to_branch: Option<String>,
+
+    /// Upgrade tag for testing, contains timestamp.
+    #[clap(global = true, hide = true, long)]
+    pub upgrade_tag_for_test_envt: Option<String>,
```

Comment:
Redundant to have the 'upgrade' prefix in arguments meant for upgrade subcommands. Consider renaming the flags..